velox/docs/functions/spark/string.rst

```diff
@@ -197,7 +197,8 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
 .. spark:function:: split(string, delimiter) -> array(string)
 
-    Splits ``string`` on ``delimiter`` and returns an array. ::
+    Splits ``string`` on ``delimiter`` and returns an array.
+    The delimiter is any string matching regex, supported by re2. ::
```

Comment:
Please also update the documentation for limit if it is added. Some draft: https://github.com/facebookincubator/velox/pull/8825/files#diff-945c1e481b55ff702e669348c241946c1b7851d575913d1a0da54952750f96daR193-R199.

Reply:
updated, thanks